warmup.py

```diff
@@ -240,17 +247,42 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if (self.rmax > (L / 2)):
+            raise ValueError('rmax > L/2')
+        else:
+            dist = np.zeros(sum(range(len(points))))                                                               #array of distance magnitudes between all point combinations
```

Comment:
This essentially builds a matrix of all pair distances. If the number of points is large, this will quickly run out of memory. We will want to find a way to rewrite this so that you process just one pair at a time. I recommend the following structure:
1. Loop over all particles i.
2. Loop over all unique pairs (i,j) involving i. This effectively means that you need j > i.
3. Compute the distance between i and j subject to the PBCs.
4. Figure out which bin this distance lies in, and add 2 to the bin (counting both the (i,j) and (j,i) pair). Skip the pair if this distance is less than rmin or greater than rmax.
5. Normalize these counts using the bin widths, then accumulate them into `self._rdf`. Increment the number of samples that is being averaged in `self._rdf`.